Source/WebKit/UIProcess/WebPageProxy.messages.in

```diff
@@ -556,7 +556,7 @@ messages -> WebPageProxy {
 #endif
 
 #if ENABLE(WEB_AUTHN)
-    SetMockWebAuthenticationConfiguration(struct WebCore::MockWebAuthenticationConfiguration configuration);
+    [EnabledBy=WebAuthenticationEnabled] SetMockWebAuthenticationConfiguration(struct WebCore::MockWebAuthenticationConfiguration configuration);
```

Comment:
I think we may want to guard it by `AllowTestOnlyIPC`, as it should not be used outside of testing.

Reply:
I don't think AllowTestOnlyIPC is the right flag as this is for functions that are called by the TestIPC Bindings